content/altinn-studio/getting-started/app-dev-course/modul2/_index.nb.md

```diff
@@ -35,36 +35,34 @@ Datamodellen definerer hvilke data man forventer å samle inn, og hvilket format
 For et enkelt skjema er det ofte en 1-1 knytning fra felter i datamodellen til felter i skjemaet, mens for mer avanserte
 skjemaer kan datamodellen og skjemaet være ganske forskjellige.
 
-En datamodell med noen eksempel-felter følger med når man oppretter en tjeneste. Man kan redigere feltnavnene
+En datamodell med noen eksempelfelter følger med når man oppretter en tjeneste. Man kan redigere feltnavnene
```

Comment:
Det følger med en datamodell med noen eksempelfelter  når du oppretter en tjeneste. Du kan kan redigere feltnavnene for å tydelig vise hva de representerer, og legge til egne felter.